apps/zotonic_mod_zotonic_site_management/priv/skel/blog/src/SITE_sup.erl

```diff
@@ -0,0 +1,50 @@
+%% @author %%FULLNAME%%
+%% @copyright %%YEAR%% %%FULLNAME%%
+%% Generated on %%DATE%%
+%% @doc %%SITE%% root supervisor.
+
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+
+-module(%%SITE%%_sup).
+-author("%%FULLNAME%%").
+
+-behaviour(supervisor).
+
+%% API functions
+-export([ start_link/0 ]).
+
+%% Supervisor callbacks
+-export([ init/1 ]).
+
+-define(SERVER, ?MODULE).
+
+%%%=============================================================================
+%%% API functions
+%%%=============================================================================
+-spec start_link() -> supervisor:startlink_ret().
+
```

Comment:
Please, remove here extra new line here or add at line 43 - just for keeping one code style.